libavfilter/vf_dnn_detect.c

```diff
@@ -48,6 +48,9 @@ typedef struct DnnDetectContext {
 #define FLAGS AV_OPT_FLAG_FILTERING_PARAM | AV_OPT_FLAG_VIDEO_PARAM
```

Comment:
is the tf2 model supported? If so, we can use that.

Reply:
what is tf2model? does it mean convert tf pb file to native model file?

Reply:
local testing was carried on with 'ssd_mobilenet_v1_coco'

why not ssd_mobilenet_v2_coco

Reply:
updated the commit message with 'ssd_mobilenet_v2_coco'